hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -121,18 +122,44 @@ public synchronized void open() throws IOException {
         String hosts = config.get(HbaseOptions.HBASE_HOSTS);
         int port = config.get(HbaseOptions.HBASE_PORT);
         String znodeParent = config.get(HbaseOptions.HBASE_ZNODE_PARENT);
-
-        Configuration hConfig = HBaseConfiguration.create();
-        hConfig.set(HConstants.ZOOKEEPER_QUORUM, hosts);
-        hConfig.set(HConstants.ZOOKEEPER_CLIENT_PORT, String.valueOf(port));
-        hConfig.set(HConstants.ZOOKEEPER_ZNODE_PARENT, znodeParent);
-
-        hConfig.setInt("zookeeper.recovery.retry",
-                       config.get(HbaseOptions.HBASE_ZK_RETRY));
-
-        // Set hbase.hconnection.threads.max 64 to avoid OOM(default value: 256)
-        hConfig.setInt("hbase.hconnection.threads.max",
-                       config.get(HbaseOptions.HBASE_THREADS_MAX));
+        boolean isEnableKerberos = config.get(HbaseOptions.HBASE_KERBEROS_ENABLE);
+        Configuration hConfig = null;
+        if(isEnableKerberos) {
+            String krb5Conf = config.get(HbaseOptions.HBASE_KRB5_CONF);
+            System.setProperty("java.security.krb5.conf", krb5Conf);
+            hConfig = HBaseConfiguration.create();
+            String principal = config.get(HbaseOptions.HBASE_KERBEROS_PRINCIPAL);
+            String keyTab = config.get(HbaseOptions.HBASE_KERBEROS_KEYTAB);
+            hConfig.set("hadoop.security.authentication", "kerberos");
```

Comment:
add hConfig.set("hadoop.security.authorization", "true");

Reply:
It's not a legal argument. After setting,program reports exception ,as follows:
Exception in thread "main" java.lang.IllegalArgumentException: Invalid attribute value for hadoop.security.authentication of true
	at org.apache.hadoop.security.SecurityUtil.getAuthenticationMethod(SecurityUtil.java:609)
	at org.apache.hadoop.security.UserGroupInformation.initialize(UserGroupInformation.java:272)
	at org.apache.hadoop.security.UserGroupInformation.setConfiguration(UserGroupInformation.java:311)